packages/harmony/src/icons/UtilityIcons.stories.mdx

```diff
@@ -35,7 +45,9 @@ Recommended sizes for utility icons: 14px | 16px | 20px | 24px | 30px | 32px
         w={iconSizes[size]}
         css={{ background: 'rgba(255, 0, 0, 0.10)' }}
       >
-        <Text css={{ color: '#F00' }}>{iconSizes[size]}</Text>
+        <Text css={{ color: '#F00 !important', fontSize: '10px !important' }}>
```

Comment:
Is this competing with something that is making these `!important`s needed?

Reply:
It's annoying, all the storybook styles are highly specific so I don't think there is another way around it?